console/backend/src/test/java/org/frankframework/console/controllers/socket/FrankApiWebSocketBaseTest.java

```diff
@@ -0,0 +1,102 @@
+package org.frankframework.console.controllers.socket;
+
+import org.frankframework.console.controllers.SpringUnitTestLocalGateway;
+
+import org.frankframework.console.controllers.WebTestConfiguration;
+import org.frankframework.console.util.RequestMessageBuilder;
+import org.frankframework.management.bus.BusAction;
+import org.frankframework.management.bus.BusTopic;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.messaging.Message;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.context.web.WebAppConfiguration;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+@WebAppConfiguration
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = { WebTestConfiguration.class, FrankApiWebSocketBase.class})
+class FrankApiWebSocketBaseTest {
+
+	@InjectMocks
```

Comment:
Is deze `@InjectMocks` en `MockitoAnnotations.openMocks(this);` echt nodig? Je initialiseert ze zelf ook al of mis ik iets?

Reply:
Dat klopt, het was een overblijfsel van iets dat ik eerder had geprobeerd